src/ArrayInterface.jl

```diff
@@ -44,6 +44,13 @@ can_setindex(x) = true
 can_setindex(x::AbstractArray) = can_setindex(typeof(x))
 can_setindex(::Type{<:AbstractRange}) = false
 
+"""
+    aos_to_soa(x)
+
+Converts an array of structs formulation to a struct of array
+"""
+aos_to_soa(x) = x
```

Comment:
I'm a bit late on this, but `aos_to_soa` does not read that well, two not-that-known abbreviations in an exported function

Reply:
Yeah, but anything I could think of was excessively long, so 🤷‍♂

Reply:
`structarray_to_arraystruct`? `rowarray_to_colarray`? There should be a similar name in the data ecosystems

Reply:
`rowarray_to_colarray` doesn't work since it really is more general structs. `structarray_to_arraystruct` would be possible.

Reply:
alternatively, writing one of them in full letter and the second in abbreviation also works, since they are redundant, `structarray_to_aos`?

Reply:
ehh I like the symmetry